packages/data-models/tests/treebank_data_item.test.js

```diff
@@ -27,6 +27,10 @@ describe('treebank_data_item.test.js', () => {
 
   it('1 TreebankDataItem - hasTreebankData is falsy if some data is missing', () => {
```

Comment:
I think we need tests for the new tb_doc, tb_sent and tb_word attributes too -- separate from the tests on tb_ref

Reply:
Agree, will add them tomorrow